frontend/src/metabase/admin/upsells/components/UpsellGem.tsx

```diff
@@ -1,7 +1,10 @@
 import { FixedSizeIcon, type IconProps } from "metabase/ui";
 
-import { upsellColors } from "./Upsells.styled";
-
 export const UpsellGem = (props: Omit<IconProps, "name" | "color">) => (
-  <FixedSizeIcon size={16} name="gem" color={upsellColors.gem} {...props} />
+  <FixedSizeIcon
+    size={16}
+    name="gem"
+    color="var(--mb-color-upsell-gem)"
```

Comment:
Is this ok or should we define a CSS rule in `Upsells.module.css` instead?

Reply:
Great, thanks.
I went a step further in order to remove the reliance on the styled component (FixedIcon)
https://github.com/metabase/metabase/pull/53727/commits/dfe59f929ed5dcf912f22cdd4b4028d5a9c780a6